website/static/js/pages/base-page.js

```diff
@@ -58,6 +58,11 @@ var SlideInViewModel = function (){
     };
     // Google Analytics click event tracking
     self.trackClick = function(source) {
+        if (source === 'Create Account') {
+            window.location = '/#signUpScopeTop';
```

Comment:
addressed in 8582919. use window.location instead of href is because that the onlick action is binded to trackClick function which ignores the href redirect.